python/GafferUI/PathPlugValueWidget.py

```diff
@@ -93,9 +93,10 @@ def getToolTip( self ) :
 
 		result = GafferUI.PlugValueWidget.getToolTip( self )
 
-		result += "\n\n<ul>"
-		result += "<li>Tab to auto-complete</li>"
-		result += "<li>Cursor down to list</li>"
+		result += "<ul>"
+		result += "<li><kbd>Tab</kbd> to autocomplete path component</li>"
+		result += "<li>Select path component (or hit <kbd>&darr;</kbd>) to show path-level contents menu</li>"
```

Comment:
Should we document that you select the path component with a double click, and the that select all is a triple click? Or is that common knowledge?

Reply:
Definitely not common knowledge. However, since you can also use <kbd>Shift</kbd> and the arrow keys, and I didn't want to run through every way one could select text in a field, I settled on "select." (As an aside, in my experience, "triple-click" isn't a phrase users encounter often, especially with text fields. "Select all"/"highlight all" is much more conventional.)

I'll switch the verb to "highlight" and "highlight all" to communicate that better. Is that enough, do you think?

Reply:
Thanks for the explanation - makes sense. I'm happier with "select" vs "highlight" I think, so I'll merge...